cylc/flow/task_proxy.py

```diff
@@ -247,6 +247,15 @@ def __str__(self):
         """Stringify using "self.identity"."""
         return self.identity
 
+    def __repr__(self):
```

Comment:
Irrelevent. I added this to help me debug. Tell me if you want it removed.

Reply:
Best to remove if not used anywhere.  (As an aside, I'd use a debugger to see this sort of thing, without the need to add anything to the code).